src/data-model-providers/codegen/CodegenDataModelProvider.cpp

```diff
@@ -159,6 +159,17 @@ std::optional<DataModel::ActionReturnStatus> CodegenDataModelProvider::Invoke(co
                                                                               TLV::TLVReader & input_arguments,
                                                                               CommandHandler * handler)
 {
+    // Some commands are registered of ALL endpoints, so make sure first that
```

Comment:
This still leaves Accepted/Generated command listing producing inconsistent results between the CHI and non-CHI cases, no?  Shouldn't we be fixing that in there?  Because we have existing logic to not even reach this point if the command path is not supported, except we skip that logic for CHI for some reason.

Reply:
Fair. Added tests in the accepted/generated command listing as well as unit test for that.